wiki/People/Beatmap_Nomination_Group/en.md

```diff
@@ -9,33 +9,33 @@
 [Ym]: /wiki/shared/Insane-m.png "Yes"
 
 Beatmap Nomination Group
-=========================
+========================
 
-The **Beatmap Nomination Group**, more commonly referred to as BN, is a group in charge of [beatmap nomination](/wiki/Beatmaps).
+The **Beatmap Nomination Group**, more commonly referred to as BNs, is a group in charge of [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure).
```

Comment:
If the group is abbreviated as the BNG, and members abbreviated as BNs, then wouldn't it make sense to say something like this?:
"The **Beatmap Nomination Group**, more commonly referred to as the BNG, is a group in charge of beatmap nominations. Individual members are commonly referred to as BNs, which stands for Beatmap Nominators.